3542/1/lib/util.rb

```diff
@@ -0,0 +1,13 @@
+class Util
```

Comment:
I don't like the way you store this method in one class.
I would create module `Util` with class `File` which respond to open file.
In the module `Util` I would create sub module `Parse` which with classes `HTML` and `YAML`.
This will at least show different responsibilities of `Util`.
```
Util::File.open(...)
Util::Parse::YAML.parse(...)
Util::Parse::HTML.parse(...)
```